templates/taxi/manufacturer_list.html

```diff
@@ -23,6 +25,12 @@ <h1>Manufacturer List
               <td>
                   {{ manufacturer.country }}
               </td>
+              <td>
+                <a href="{% url 'taxi:manufacturer-update' pk=manufacturer.id %}">Update</a>
+              </td>
+              <td>
+                <a style="color: red" href="{% url 'taxi:manufacturer-delete' pk=manufacturer.id %}">Delete</a>
+              </td>
```

Comment:
You can also add names of delete/update columns

Reply:
I don't agree.. In my opinion it is unnecessary to add names for columns which consists of only one element

Reply:
but okey) I'll add columns names =))